### DISCUSSION TITLE
npm run dev --port 8080  npm ERR! docs@4.8.0 dev: `concurrently -n "server,theme-server" -c "bgMagentaBright.bold,bgGreenBright.bold" "npm run dev:server" "npm run dev:theme-server" "8080"`

### DISCUSSION THREAD

**Discussion by jclivi** [rsuite/rsuite#1507] [General]
### Describe the Bug

npm ERR! docs@4.8.0 dev: `concurrently -n "server,theme-server" -c "bgMagentaBright.bold,bgGreenBright.bold" "npm run dev:server" "npm run dev:theme-server" "8080"`

**Comment by simonguo** [comment]
Can you provide more detailed error information?
